docs-ref-services/servicebus.md

```diff
@@ -2,47 +2,205 @@
 
-Microsoft Azure Service Bus supports a set of cloud-based, message-oriented middleware technologies including reliable message queuing and durable publish/subscribe messaging. 
+* [SDK source code](https://github.com/Azure/azure-sdk-for-python/tree/master/azure-servicebus)
+* [SDK reference documentation](https://docs.microsoft.com/python/api/overview/azure/servicebus/client?view=azure-python)
 
-## Install the libraries
+## What's new in v0.50.0?
+As of version 0.50.0 a new AMQP-based API is available for sending and receiving messages. This update involves **breaking changes**.
+Please read [Migration from v0.21.1 to v0.50.0](#migration-from-v0211-to-v0500) to determine if upgrading is
+right for you at this time.
+
+The new AMQP-based API offers improved message passing reliability, performance and expanded feature support going forward.
+
+For documentation on the legacy HTTP-based operations please see [Using HTTP-based operations of the legacy API](#using-http-based-operations-of-the-legacy-api).
+
+
+## Prerequisites
+
+* Azure subscription - [Create a free account](https://azure.microsoft.com/free/)
+* Azure [Service Bus namespace and management credentials](https://docs.microsoft.com/azure/service-bus-messaging/service-bus-create-namespace-portal)
+* [Python 2.7-3.7](https://www.python.org/downloads/)
+
+
+## Installation
 ```bash
 pip install azure-servicebus
 ```
 
-## ServiceBus Queues
-ServiceBus Queues are an alternative to Storage Queues that might be
+## Connect to Azure Service Bus
+
+### Get credentials
+Use the Azure CLI snippet below to populate an environment variable with the Service Bus connection string (you can also find this value in the Azure portal). The snippet is formatted for the Bash shell.
+
+```Bash
+RES_GROUP=<resource-group-name>
+NAMESPACE=<servicebus-namespace>
+
+export SB_CONN_STR=$(az servicebus namespace authorization-rule keys list \
+ --resource-group $RES_GROUP \
+ --namespace-name $NAMESPACE \
+ --name RootManageSharedAccessKey \
+ --query primaryConnectionString \
+ --output tsv)
+```
+
+### Create client
+Once you've populated the `SB_CONN_STR` environment variable, you can create the ServiceBusClient.
+```python
+import os
+from azure.servicebus import ServiceBusClient
+
+connection_str = os.environ['SB_CONN_STR']
+
+sb_client = ServiceBusClient.from_connection_string(connection_str)
+```
+
+
+## Service Bus Queues
+Service Bus Queues are an alternative to Storage Queues that might be
 useful in scenarios where more advanced messaging features are needed
 (larger message sizes, message ordering, single-operation destructive
 reads, scheduled delivery) using push-style delivery (using long
 polling).
 
-The service can use Shared Access Signature authentication, or ACS
-authentication.
+The service can use Shared Access Signature authentication.
```

Comment:
This line is a bit out of place now since it looks like SAS is the only auth method. It's standing alone makes it a call-out, but there is no other follow-up text describing or calling out that auth method until much further down in the migration section.

You could remove the line altogether, or expand on it by changing it to "The service uses Shared Access Signature authentication." and adding another sentence or two describing SAS, optionally linking to SAS auth for Service Bus content elsewhere.